### PR TITLE
Implemented `hit fix` [WIP]

### PR DESCRIPTION
**Pull request by Jigar3**
@ChShersh Will rectify the duplication in code.

### CODE REVIEW

File: src/Hit/Git.hs [hunk 1]
```diff
@@ -72,6 +73,18 @@ runCommit (T.strip -> msg) (not -> hasIssue)
            let issue = "#" <> show n
            in "[" <> issue <> "] " <> msg <> "\n\nResolves " <> issue
 
+-- / @hit fix@ command
+runFix :: Maybe Text -> IO ()
+runFix = \case
+    Nothing -> do
+        "git" ["add", "."]
+        "git" ["commit", "-m", "Fix after review"]
```
Comment (vrom911) [inline src/Hit/Git.hs]: I guess you can find [`fromMaybe`](https://hackage.haskell.org/package/relude-0.4.0/docs/Relude-Monad-Reexport.html#v:fromMaybe) useful in here. With it, you won't need to pattern match on the `Maybe Text` :slightly_smiling_face:

**Comment by Jigar3** [comment]
@ChShersh @vrom911 Hopefully I have done it correctly. :smiley:

**Comment by Jigar3** [comment]
@vrom911 @ChShersh Can you give me one more issue which I could solve??

**Comment by ChShersh** [comment]
@Jigar3 You can look at open issues for `hit-on` and see whether there's something interesting for you :slightly_smiling_face:

**Pull request merged**